### ISSUE TITLE
conda-forge package out-dated

### ISSUE THREAD

**Issue by MarekOzana** [matthewgilbert/blp#39]
the latest available pypi package is 0.0.3 while the latest conda-forge package is out-dated 0.0.2 (missing bql functionality)

#### Code Sample, a copy-pastable example if possible

```python
>>> pip install blp 
# Installing collected packages: blp
# Successfully installed blp-0.0.3

>>> conda install blp
# The following NEW packages will be INSTALLED: blp
#                //noarch::blp-0.0.2-pyhd8ed1ab_0

```
#### Problem description

When installing blp using conda, one gets out-dated version 0.0.2 without bql

#### Expected Output

Update conda-forge package to 0.0.3

**Comment by matthewgilbert** [comment]
Just merged https://github.com/conda-forge/blp-feedstock/pull/2 so this should kick off a build and update on conda-forge

**Comment by MarekOzana** [comment]
> Just merged [conda-forge/blp-feedstock#2](https://github.com/conda-forge/blp-feedstock/pull/2) so this should kick off a build and update on conda-forge

Thanks @matthewgilbert !
I can confirm conda install now works great.
